treeherder/intermittents_commenter/commenter.py

```diff
@@ -229,32 +214,30 @@ def new_request(self):
         }
         return session
 
-    def create_url(self, bug_id):
-        return settings.BZ_API_URL + '/rest/bug/' + str(bug_id)
-
-    def fetch_bug_details(self, params, bug_id):
+    def fetch_bug_details(self, bug_ids):
         """Fetches bug metadata from bugzilla and returns an encoded
            dict if successful, otherwise returns None."""
 
+        params = {'include_fields': 'product, component, priority, whiteboard, id'}
+        params['id'] = bug_ids
         try:
-            response = self.session.get(self.create_url(bug_id), headers=self.session.headers, params=params,
-                                        timeout=30)
+            response = self.session.get(settings.BZ_API_URL + '/rest/bug', headers=self.session.headers,
+                                        params=params, timeout=30)
             response.raise_for_status()
         except RequestException as e:
-            logger.warning('error fetching bugzilla metadata for bug {} due to {}'.format(bug_id, e))
+            logger.warning('error fetching bugzilla metadata for bugs due to {}'.format(e))
             return None
 
-        # slow down: bmo server may refuse service if too many requests made too frequently
-        time.sleep(0.5)
         data = response.json()
         if 'bugs' not in data:
             return None
 
-        return {key.encode('UTF8'): value.encode('UTF8') for key, value in iteritems(data['bugs'][0])}
+        return data['bugs']
 
     def submit_bug_changes(self, changes, bug_id):
+        url = settings.BZ_API_URL + '/rest/bug/' + str(bug_id)
```

Comment:
Just a nit:  You could also use ``.format()`` here for making these strings so you don't have to worry about casting:

    url = '{}/rest/bug/{}'.format(settings.BZ_API_URL, bug_id)

Reply:
oh right! :)